### PR TITLE
Updating dependencies and tagging version 11

### LINKED ISSUE #116: Update to solr_wrapper needed
Hydra 10.0.1 asks for solr_wrapper ~> 0.15.0, but the latest release of solr_wrapper is 0.19.0. This makes the difference b/w the task in "Dive Into Hydra" working by default vs someone having to go figure out what the latest version of solr is.

I am about to fix this and issue a pull request, I just wanted to have an issue to close when I do.

### PR DESCRIPTION
**Pull request by bess**
This should fix #116 and #117. I ran through Dive Into Hydra with it and couldn't find any problems. As soon as it's released I'll update the Dive Into Hydra tutorial accordingly.

### CODE REVIEW

File: hydra.gemspec [hunk 1]
```diff
@@ -23,18 +23,16 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license = 'APACHE2'
 
-  gem.add_dependency 'fcrepo_wrapper', '~> 0.5.1'
-  gem.add_dependency 'solr_wrapper', '~> 0.15.0'
-  gem.add_dependency 'active-fedora', '~> 10.0.0'
-  gem.add_dependency 'hydra-head', '~> 10.0.0'
-  gem.add_dependency 'rails', '~> 4.2'
-  gem.add_dependency 'om', '~> 3.1.1'
+  gem.add_dependency 'fcrepo_wrapper', '~> 0.7'
```
Comment (jcoyne) [inline hydra.gemspec]: The idea with the hydra gem was to tightly pin the dependencies, such that they wouldn't change when you do `bundle update`.  That why the previous version was pinned pessimistically. This allowed you to get bug fixes, but not new features.  I think still makes sense to continue that practice. What do you think?

Comment (bess) [inline hydra.gemspec]: Oh, sure, I didn't understand that was a feature not a bug. Sure, I'll update.

Comment (jcoyne) [inline hydra.gemspec]: Maybe documenting this in a comment would be a kindness to future maintainers.

**Comment by bess** [comment]
@jcoyne Is this what you meant?

File: hydra.gemspec (hunk 1 above)
Comment (bess) [inline hydra.gemspec]: Oh good point, I'll add that too. Dang, sorry I didn't see this before my last commit.

File: lib/hydra/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Hydra
-  VERSION = "10.0.1".freeze
+  VERSION = "11.0.0".freeze
```
Comment (jcoyne) [inline lib/hydra/version.rb]: Looks good.  Do you want to make this 11.0.0.rc1 or 11.0.0.beta first? Or are you confident it's GTG?

Comment (bess) [inline lib/hydra/version.rb]: Happy to make it rc1. Not sure what the QA process on this is. I've run through the tutorial with it but I'd be happy to see more testing done.

**Pull request merged**